.github/workflows/pr-test.yml

```
@@ -0,0 +1,44 @@
+name: Test with Gradle
+on:
+  pull_request:
+    branches: dev
+permissions:
+  contents: write
+  pull-requests: write
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)